fe1-web/src/features/evoting/screens/CreateElection.tsx

```diff
@@ -46,6 +47,16 @@ const EMPTY_QUESTION: NewQuestion = {
 const MIN_BALLOT_OPTIONS = 2;
 
 const styles = StyleSheet.create({
+  view: {
+    ...Views.base,
+    flexDirection: 'row',
+    zIndex: 3,
+  } as ViewStyle,
+  viewVertical: {
+    ...Views.base,
+    flexDirection: 'column',
+    zIndex: 3,
+  } as ViewStyle,
```

Comment:
Maybe this style relative to events could be stored somewhere instead of being copy/pasted 3 times.

Reply:
Yes that is true but I plan to change the UI a bit around next week so I thought I would tackle this then 😅 But since it is not much effort I'll quickly do it :)